Update variant_pricing tag documentation (#237)
easolhq/easolhq.github.io: docs/reference/tags/variant_pricing/index.md

    })

{% endraw %}

## Consideration with packages

> When this tag is used in the context of a package step, the user may need to
> choose an exact time slot for the variant after choosing the variant. In this case
> we calculate the cheapest possible price of the possible slots. We recommend
> pre-pending the price with 'From' in this case.
> 

Reviewer: ```suggestion
When this tag is used in the context of a package step, the user may need to
choose from a range of time slots. In this case we calculate the cheapest possible price of the possible slots. We recommend pre-pending the price with 'From' in this case.
```
